doc/release-notes/release-notes-6648.md

```diff
@@ -0,0 +1,4 @@
+GUI changes
+--------
+
+- A new menu item to restore a wallet from a backup file has been added (#6648).
```

Comment:
471: note: partial release notes used to go into `doc/`, not into `doc/release-notes/`, but 471 is doing it differently.